concent_api/core/message_handlers.py

```diff
@@ -1305,9 +1312,13 @@ def handle_send_subtask_results_verify(
 
 def handle_message(client_message):
     if isinstance(client_message, message.ForceReportComputedTask):
+        assert isinstance(client_message.report_computed_task.task_to_compute.requestor_public_key, str)
```

Comment:
remove those (I mean all new in this function) asserts, they are reduntdant - in each handler there is `validate_task_to_compute` function that takes care of that